apps/website/src/_state/component-statuses.ts

```diff
@@ -27,6 +27,7 @@ export const statusByComponent: ComponentKitsStatuses = {
     RadioGroup: ComponentStatus.Draft,
     Separator: ComponentStatus.Beta,
     Skeleton: ComponentStatus.Beta,
+    Sonner: ComponentStatus.Draft,
```

Comment:
Perhaps naming this Toast? I know that Shadcn calls this sonner, but I believe that's because they already have a toast component.

Might be a bit easier to understand for consumers.

Reply:
I believe shadcn calls Sonner because the headless library was created with the name Sonner (https://sonner.emilkowal.ski/).

But, **I agree with you**, we can change to Toast and with this, we don't need to rush to have a headless Toast component from qwik-ui.

I will update.

Reply:
Changed.

Reply:
> I believe shadcn calls Sonner because the headless library was created with the name Sonner (https://sonner.emilkowal.ski/).
> 
> But, **I agree with you**, we can change to Toast and with this, we don't need to rush to have a headless Toast component from qwik-ui.
> 
> I will update.

Yeah I'd be curious to know if they'd still call it sonner if there wasn't a toast component they already had. I'll bet the didn't wanna call it Toast (2) haha.